packages/front-end/components/Experiment/ResultsDownloadButton.tsx

```diff
@@ -86,7 +86,7 @@ export default function ResultsDownloadButton({
             perUserValueStdDev: stats.stats.stddev || null,
             chanceToBeatControl: stats.chanceToWin || null,
             percentChange: stats.expected || null,
-            percentChangePValue: stats.pValue || null,
+            percentChangePValue: stats.pValue ?? null,
```

Comment:
should the surrounding fields also have `||` instead of `??` ?
`perUserValueStdDev`, `chanceToBeatControl`, `percentChange`, `percentChangeCILower`, `percentChangeCIUpper`?

Reply:
I also considered changing them, but didn't for two reasons:
1) they are far less likely to be very, very small like p-values, so unlikely to be meaningfully 0
2) the defaults we set for them in various places are 0 when they are in fact missing.

An exception to the above is `stats.chanceToWin` which defaults to 50 in these cases. So I would update that field to `??` and will do so now.